Formula/util-linux.rb

```diff
@@ -11,7 +12,7 @@ class UtilLinux < Formula
     sha256 "02c15639cc3e40c7553dbcb23e94a5e679feeaad8881ed6ddce91c4cb0a015b2" => :sierra
   end
 
-  conflicts_with "rename", :because => "both install `rename` binaries"
+  keg_only :provided_by_macos, "macOS provides the uuid.h header"
```

Comment:
Not really provided by macOS in this case as much as a single header is?

Reply:
Well macOS provides both header and libuuid, which is compiled into libSystem.dylib. I agree that util-linux provides other things as well, that are not provided by macOS, but I don't see what else I can do to get rid of the audit error.

Reply:
Remove `:provided_by_macos, `

Reply:
Done